pandas/tests/test_groupby.py

```diff
@@ -4594,9 +4594,6 @@ def test_frame_groupby_plot_boxplot(self):
         _skip_if_mpl_not_installed()
 
         import matplotlib.pyplot as plt
-        import matplotlib as mpl
```

Comment:
Euhm, I would keep it here, as it really tests the plotting method of the GroupBy object. But maybe you could see if you can eg put the `_skip_if_mpl_not_installed` in util.testing for reuse.